### ISSUE TITLE
Does caster consume data from server when no clients connected?

### ISSUE THREAD

**Issue by d1ken** [Node-NTRIP/caster#1]
Hello.
Does caster consume data from server when no clients connected to mountpoint?
As I understand stream buffering mechanism, it should not.

**Comment by d1ken** [comment]
OK, found statsStream.
How to stop it from consumption data all the time.
I think better to set some consumption interval when no clients connected.
Something like 30 second per hour or more - should receive all RTCM messages with more that 1 sec rate (1005, 1230).

**Comment by nebkat** [comment]
That is correct, `statsStream` currently consumes the data when no clients are connected. I think if this sort of function is to exist it should be one of the mountpoint options and be configurable, either as you say based on message rate or manually, and for information purposes a most recent message of type map should exist anyway. I'll consider adding but it would be useful to know which messages can/can't be sent delayed.
